drafts/inlinemacros.md

```diff
@@ -35,8 +34,12 @@ members. Instead, every inline member becomes an overloaded alternative
 of all other members with the same name. Normal overloading resolution
 is applied to pick an inline member over some other member.
 
-Inline definitions only exist at compile-time; no storage is allocated for them and
-no code is generated for them. That means that it is OK to have an inline member that
+If an inline definition doesn't explicitly specify its result type,
+the result type gets inferred according to the usual rules.
+
+Inline definitions only exist at compile time; no storage is allocated for them in object layout
+and no code is generated for them in object method tables.
```

Comment:
Just reworded the original formulation. One might argue that inline definitions have storage allocated for them in TASTY and *byte*code generated for them in classfiles, so I tried to make this a bit more careful. I'm not very happy with the result, though.